sass/components/_button.scss

```diff
@@ -40,10 +40,20 @@
     box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 0px 4px 30px #ff0000;
   }
 
-  &--correctAnswer {
+  &--answer:hover {
+    background-color: $color-bgc;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
+      inset 0px 4px 4px rgba(0, 0, 0, 0.25);
+  }
+
+  &--correctAnswer:focus {
+    font-weight: 600;
     background-color: #41ed25;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 0px 4px 30px #51fc00;
   }
-  &--incorrectAnswer {
+  &--incorrectAnswer:focus {
+    font-weight: 600;
     background-color: #ff0000;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 0px 4px 30px #ff0000;
```

Comment:
Ustawienie focus pozwala łatwo oszukać w grze.
Przytrzymasz myszą i widać która odpowiedź, a nie jest liczone kliknięcie

Reply:
Aaa no tak, bo focus nawet tabem chyba można. Nie wiem, czy nie trzeba w JSie jednak po kliknięciu dodać jakiejś klasy np. Ja słaby jestem z CSSa - może da się to jakoś osiągnąć stylem?

Reply:
Tabem chyba też.
Dodatkowo Po kliknięciu na odpowiedź i kliknięciu w innym miejscu ekranu odpowiedź odznaczy się.
Plus focusem zaznaczymy chyba tylko jedna odpowiedź na raz.
Dodalem w Btn dodawanie klasy przez wywolanie funkcji. (ale to w PR poprawkowym)

Reply:
:visited będzie ok?

Reply:
```CSS-owy znacznik :visited pozwalał stronom WWW na ujawnianie historii przeglądania użytkownika i które ze stron odwiedził.```

Też odpada.
Po odpaleniu gry 2 raz będzie już wszystko wiadomo.
On nie do tego służy.  Samym CSS bez JS (np dodawanie klas przez kod js) nie obsłuzymy tego poprawnie.